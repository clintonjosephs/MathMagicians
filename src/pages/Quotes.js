import React from 'react';

const Quotes = () => (
  <div style={{ margin: '10% auto', width: '70%' }}>
    <h2 className="h2-general">Random Quotes</h2>
    <div className="text-wrapper">
      Mathematics is not about numbers, equations, computations or algorithms:
      is is about understanding. -William Paul Thurston
    </div>
  </div>
);

export default Quotes;
