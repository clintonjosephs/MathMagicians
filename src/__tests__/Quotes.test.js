import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../pages/Quotes';

describe('Use Jest snapshots to test the Quotes.', () => {
  it('Renders Quotes component', () => {
    const component = renderer.create(
      <Quotes />,
    ).toTree();
    expect(component).toMatchSnapshot();
  });
});
