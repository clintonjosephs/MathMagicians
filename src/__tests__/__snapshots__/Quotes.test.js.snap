// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use Jest snapshots to test the Quotes. Renders Quotes component 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="h2-general"
        >
          Random Quotes
        </h2>,
        <div
          className="text-wrapper"
        >
          Mathematics is not about numbers, equations, computations or algorithms: is is about understanding. -William Paul Thurston
        </div>,
      ],
      "style": Object {
        "margin": "10% auto",
        "width": "70%",
      },
    },
    "rendered": Array [
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": "Random Quotes",
          "className": "h2-general",
        },
        "rendered": Array [
          "Random Quotes",
        ],
        "type": "h2",
      },
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": "Mathematics is not about numbers, equations, computations or algorithms: is is about understanding. -William Paul Thurston",
          "className": "text-wrapper",
        },
        "rendered": Array [
          "Mathematics is not about numbers, equations, computations or algorithms: is is about understanding. -William Paul Thurston",
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  "type": [Function],
}
`;
